keras_hub/src/models/stable_diffusion_3/stable_diffusion_3_presets.py

```diff
@@ -43,6 +43,6 @@
             "path": "stable_diffusion_3",
             "model_card": "https://huggingface.co/stabilityai/stable-diffusion-3.5-large-turbo",
         },
-        "kaggle_handle": "kaggle://kerashub/stablediffusion3/keras/stable_diffusion_3.5_large/1",
+        "kaggle_handle": "kaggle://keras/stablediffusion-3.5/keras/stable_diffusion_3.5_large/1",
```

Comment:
I made a mistake here:
For turbo version, the `kaggle_handle` should be `"kaggle://keras/stablediffusion-3.5/keras/stable_diffusion_3.5_large_turbo/1"` which corresponds to `"kaggle://kerashub/stablediffusion3/keras/stable_diffusion_3.5_large_turbo/1"`